build-logic/packaging/src/main/kotlin/gradlebuild.instrumentation-metadata.gradle.kts

```diff
@@ -33,10 +33,14 @@ val extension = extensions.create<InstrumentationMetadataExtension>("instrumenta
     classpathToInspect = files()
     superTypesOutputFile.convention(layout.buildDirectory.file("instrumentation/instrumented-super-types.properties"))
     superTypesHashFile.convention(layout.buildDirectory.file("instrumentation/instrumented-super-types-hash.txt"))
+    upgradedPropertiesFile.convention(layout.buildDirectory.file("instrumentation/upgraded-properties.json"))
+    upgradedPropertiesHashFile.convention(layout.buildDirectory.file("instrumentation/upgraded-properties-hash.txt"))
```

Comment:
🤔 It seems redundant that we specify both a convention and set the values explicitly in `gradlebuild.distributions.gradle.kts`. Can we lose the conventions?

This is a single use plugin for our own build, not a general-purpose one used in many different contexts. As such, defining a convention that we override anyway doesn't make much sense to me.

Reply:
Done